docs/adr/0001-daily-active-use-server-side-metrics-glean.md

```diff
@@ -0,0 +1,121 @@
+
+The Glean team does/did not have capacity to implemented the Rust `glean_parser` feature, so we had to decide what is the best solution not only for this use case, but to consider possible future use cases. In consultation with the Glean team, it became clear avoiding our custom implementation and opting for the general-purpose `glean_parser` for Rust was the ideal solution.
+
+## Pros and Cons of the Options
+
+### A. Glean Parser for Rust
+
+Glean is a widely used tool at Mozilla and provides us with a solution to the given issue and possible extensibility in the future. Not without some challenges in initial implementation, related to coordination with Glean team and upfront development effort. However, the potential for positive impact within our team and the organization is significant: all Rust server applications will be able to use Glean with full server support, our possible intention to integrate Glean into Push is made easier, and this is done in partnership with the Glean team.
+
+#### Pros
+
+* Makes Glean compatible for all Rust server applications going forward.
+* Preferred option of the Glean team.
+* Glean team believes, based on FxA metrics volume, that our volume will not be a problem (180-190K per minute).
+* A collection of metrics, emitted as a single "Ping Event" make querying of related data simpler.
+* Core of Glean's purpose is to measure user interactions and the rich metadata that accompanies it.
+* Capacity for future expansion of application metrics within Sync, beyond DAU.
+* Prepares for implementation of same measurements in autopush, also using Glean.
+* Easier to query, have data team support to set up queries.
+* Use of standardized Mozilla tooling.
+* Establishment of team knowledge of using Glean.
+* Establishment of server-side Rust best practices, leading to easier development for backend Rust applications.
+* Transparency in data review process with consideration made to minimum collection of data.
+
+#### Cons
+
+* The `glean_parser` tool used by other Glean applications is currently not supported for Rust. Furthermore, client applications can use the Glean SDK and this is also not an option for us.
+* Server side metrics have not yet been implemented for a Rust server application of this kind, so this is new territory.
+* There is added complexity of data review process and registration of the application to Glean's probe scraper, PubSub topics, ingestion pipeline etc.
+* May require more uptfront development effort to get up to speed with new code and process.
+* Some concerns exist around volume of data emitted from the service and if it is feasible, but we won't know until we try.
```

Comment:
Is this last bullet still a concern? Earlier we said that the Glean team was confident. If there's a risk here, we should state it -- like if we're wrong about the volume -- but if we are also confident in the volume estimate given to the Glean team, maybe we don't need this bullet.

Reply:
Originally it may have been a concern, but from the Glean team's point of view (and the volume related to FxA) we're probably good, however we really won't know until we see the output. I think this warrants removing this point.